Source/Scene/SkyAtmosphere.js

```diff
@@ -111,6 +114,9 @@ define([
             },
             fScaleOverScaleDepth : function() {
                 return (1.0 / (that._outerRadius - innerRadius)) / rayleighScaleDepth;
+            },
+            iDayNight : function() {
```

Comment:
If you want to take this to the next level (it will be easy!)
* Remove the Hungarian Notation.  This is some of the oldest code in Cesium and I just used the same names from the GPU Gems 2 chapter this is based on.
* Combine the uniforms into fewer uniforms for better performance, e.g., four floats into a `vec4` like [here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Shaders/BillboardCollectionVS.glsl#L4-L11).  Some of these - that we are likely to change - could also be constants in the GLSL.  @lilleyse can advise.